ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml

```diff
@@ -30,15 +30,39 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: ocp-qe-perfscale-ci-tests-aro-cluster-density-v2
+- as: ocp-qe-perfscale-ci-tests-node-density-cni-6nodes-aro
```

Comment:
@vishnuchalla Is the intention here not to run cluster-density-v2 tests for ARO? Hence, it's deleted? 
Do we want to run only node-density tests?

Reply:
@vishnuchalla IMO, we should still keep cluster-density-v2 tests as they are more simplistic and reliable for catching any infra issues quicker than node-density tests.

Reply:
Ack. Reg:cluster-density v2 tests, we have this [chain](https://steps.ci.openshift.org/chain/openshift-qe-ocp-qe-perfscale-ci-tests) included, that covers cluster-density-v2.